src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperOutput.cs

```diff
@@ -33,12 +34,18 @@ internal TagHelperOutput(string tagName)
         /// <param name="tagName">The HTML element's tag name.</param>
         /// <param name="attributes">The HTML attributes.</param>
         public TagHelperOutput(string tagName, [NotNull] IDictionary<string, string> attributes)
+            : this(tagName, attributes, HtmlEncoder.Default)
```

Comment:
With how we've designed `TagHelper`s users/authors can be the same; we've made them simple enough to be used as enhancements to the "average" application. For `TagHelper` framework authors such as ourselves if we ever plan to `new` up `TagHelperOutput`s then we'd forcefully set the HTMLEncoder, much like we do for `string.Format` when providing the culture. Ultimately we have the option for when it makes sense otherwise it's still easy.